src/layout/main.jsx

```diff
@@ -30,18 +31,18 @@ const Layout = ({ children }) => {
     }
   `);
   return (
-    <>
+    <GlobalProvider>
```

Comment:
This makes a lot of sense, good stuff man. It. is. generally a good idea to have adistinction between page components (templates), layouts,  and regular components, so this is a very good update.This also should enable you to work with the theme without any problems. Keep in ming the theme needs to be. wrapping any styled components that need to use. it, so having it as. the. first thing in the layout makes sense :)